resources/lib/metadata.py

```diff
@@ -192,13 +194,15 @@ def get_properties(self, api_data):
 
                 position = self._resumepoints.get_position(asset_id)
                 total = self._resumepoints.get_total(asset_id)
+                # Master over Kodi watch status
                 if position and total and SECONDS_MARGIN < position < total - SECONDS_MARGIN:
                     properties['resumetime'] = position
+                    properties['totaltime'] = total
                     log(2, '[Metadata] manual resumetime set to %d' % position)
-
-            duration = self.get_duration(api_data)
-            if duration:
-                properties['totaltime'] = duration
+                else:
+                    # Slave to Kodi watch status
+                    properties['resumetime'] = -1
```

Comment:
What is the benefit of doing this over not setting it ?
Also, I wasn't sure if setting the totaltime here has any value.

Reply:
You're right I think, I'll remove this.